src/index.js

```diff
@@ -54,8 +54,7 @@ module.exports = (
       type: "filesystem",
       cacheDirectory: typeof cache === 'string' ? cache : nccCacheDir,
       name: "ncc",
-      version: require('../package.json').version,
-      store: "instant"
```

Comment:
If switching to the `pack` mode, should we also now ensure that the cache is a per-project cache in the node_modules folder instead of the single global cache?

Reply:
That would make sense.

Another option is to keep the cache location, but use a different `cache.name` for each directory.

Like this: `name: hashOf(directory)`

Reply:
The risk with the global cache with this new storage seems to be disk usage growth right?

I guess this can be mitigated with a simple eviction strategy in due course. But that was another argument for per-project caching.

The argument for global caching is that ideally it would be great if the global cache could be smart enough to know the hashes of the dependencies (yarn.lock again!), and thus share compilations.

Not sure which of the above you want to shoot for here, but will trust your judgement!